packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -493,22 +506,124 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
       controller: _optionsViewController,
       overlayChildBuilder: _buildOptionsView,
       child: TextFieldTapRegion(
-        child: Container(
-          key: _fieldKey,
-          child: Shortcuts(
-            shortcuts: _shortcuts,
-            child: Actions(
-              actions: _actionMap,
-              child: CompositedTransformTarget(
-                link: _optionsLayerLink,
+        child: LayoutBuilder(
+          builder: (BuildContext context, BoxConstraints boxConstraints) {
+            assert(boxConstraints.hasBoundedWidth);
```

Comment:
Are there guarantees that the incoming max width will be finite?

Reply:
Not sure why I added that... I'll remove it and keep an eye out for problems with width.